src/DiscGolfScorecard1.java

```diff
@@ -0,0 +1,365 @@
+     * {@code distances}, and {@code strokes}.
+     *
+     * @param pars
+     *            array of pars in scorecard
+     * @param distances
+     *            array of distances in scorecard
+     * @param strokes
+     *            array of strokes in scorecard
+     * @requires |pars| = |distances| = |strokes| and |pars| > 0
+     * @ensures this = [default scorecard with specified pars, distances, and
+     *          strokes]
+     */
+    public DiscGolfScorecard1(int[] pars, int[] distances, int[] strokes) {
+        assert pars.length == distances.length
+                && distances.length == strokes.length : "Violation of "
+                        + "[|pars| = |distances| = |strokes|]";
+        assert pars.length > 0 : "Violation of [|pars| > 0]";
+        this.createNewRep(DEFAULT_STARTING_HOLE, pars, distances, strokes);
+    }
+
+    /**
+     * Constructor resulting in a scorecard with specified {@code pars},
+     * {@code distances}, {@code strokes}, and with startingHole
+     * {@code startingHole}.
+     *
+     * @param pars
+     *            array of pars in scorecard
+     * @param distances
+     *            array of distances in scorecard
+     * @param strokes
+     *            array of strokes in scorecard
+     * @param startingHole
+     *            starting hole in scorecard
+     * @requires @requires |pars| = |distances| = |strokes| and |pars| > 0 and 0
+     *           &lt; startingHole &le; |pars|
+     * @ensures this = [default scorecard with specified pars, distances, and
+     *          strokes]
+     */
+    public DiscGolfScorecard1(int[] pars, int[] distances, int[] strokes,
+            int startingHole) {
+        assert pars.length == distances.length
+                && distances.length == strokes.length : "Violation of "
+                        + "[|pars| = |distances| = |strokes|]";
+        assert pars.length > 0 : "Violation of [|pars| > 0]";
+        assert 0 < startingHole && startingHole <= pars.length : "Violation of "
+                + "[0 < startingHole <= |pars|]";
+    }
+
+    /*
+     * Standard methods --------------------------------------------------------
+     */
+
+    @Override
+    public final DiscGolfScorecard newInstance() {
+        try {
+            return this.getClass().getConstructor().newInstance();
+        } catch (ReflectiveOperationException e) {
+            throw new AssertionError(
+                    "Cannot construct object of type " + this.getClass());
+        }
+    }
+
+    @Override
+    public final void clear() {
+        int[] pars = new int[DEFAULT_SCORECARD_LENGTH];
+        Arrays.fill(pars, DEFAULT_PAR);
+        int[] distances = new int[DEFAULT_SCORECARD_LENGTH];
+        Arrays.fill(distances, DEFAULT_DISTANCE);
+        int[] strokes = new int[DEFAULT_SCORECARD_LENGTH];
+        Arrays.fill(strokes, DEFAULT_STROKES);
+        this.createNewRep(DEFAULT_STARTING_HOLE, pars, distances, strokes);
```

Comment:
You should probably be able to get a default implementation with just createNewRep (see my comments above).